core/src/main/java/com/github/jsonldjava/core/Context.java

```diff
@@ -1146,57 +1144,4 @@ else if (this.get(JsonLdConsts.LANGUAGE) != null) {
         }
         return rval;
     }
-
-    public Map<String, Object> serialize() {
```

Comment:
What is the reasoning behind removing this method? I am not familiar with what users have been using it for in the past, but I don't mind removing it if the reasoning is appropriate. 

It is on the Public API, so we will need a version bump to reflect the removal but I will do that after the PR is merged.

Reply:
I see now that it is in the public API. I removed it since after the changes I made it wasn't been used anywhere in the codebase and it made the coverage decrease. 

I don't know what would be the usecase for this method, since the client that makes the call had to previously create the context and so had already access to the serialized value since the way to create the `Context` is:

```java
val parsedContextValue = JsonUtils.fromString(contextValue)
val ctx = new Context(options).parse(parsedContextValue)
// parsedContextValue should be roughly the same as  ctx.serialize()
```

In any case, if you want to keep the method I don't really mind. So what's your suggestion here::
1. Keep it
2. Keep it and flag it as deprecated
3. Remove it and bump the dependencies to reflect an API change

Reply:
I would prefer to keep it and flag it as deprecated at this point. If it is being actively used by someone and they query it, we can review its deprecation at that point. Otherwise we can remove it in a future release based on deprecating it now.

Reply:
Done